tests/test_visitors/test_ast/test_loops/test_loops/test_infinite_while_loops.py

```diff
@@ -7,58 +7,77 @@
 )
 from wemake_python_styleguide.visitors.ast.loops import WrongLoopVisitor
 
-incorrect_loop1 = """
+# Wrong
+
+incorrect_loop_template_nonfunction1 = """
 while True:
     print('forever')
 """
 
-incorrect_loop2 = """
+incorrect_loop_template_nonfunction2 = """
 while True:
     while True:
         print('forever')
-        break
+        {0}
+"""
+
+incorrect_loop_template_nonfunction3 = """
+while True:
+    if some:
+        print('forever')
```

Comment:
This is not parametrized

Reply:
How can I parametrize this in a way where {0} accepts any command except for (raise, return, and break)?

Reply:
Basically my most recent pull request has (raise, return, break, etc) parametrized, but not "print('forever')" which should just be replaced by any command that is not (raise, return, break, etc).